packages/flame/src/Core/index.tsx

```diff
@@ -142,13 +142,16 @@ const FlameGlobalStyles: React.FunctionComponent<{ themeName?: string }> = ({ th
   );
 };
 
+const cssGet = (path: string) => (theme: any) => themeGet(path)({ theme });
```

Comment:
You do the same thing in #48, we're keeping this one, correct? Perhaps let's merge Switch first and update this PR afterwards.

Reply:
Similar.

The solution in #48 is a bit more elegant tho, so I'll need to rebase the changes properly with whatever I've done in #48.